foundations/installations/installations.md

```diff
@@ -261,14 +256,82 @@ If you encounter an error requesting you to disable **Intel RST** while attempti
 
 1.  You have a supported Chromebook:
-    -   [Official Chromebooks](https://www.chromium.org/chromium-os/chrome-os-systems-supporting-linux)
-    -   [ChromeOS Flex Chromebooks](https://support.google.com/chromeosflex/answer/11513094)
+    - [Official Chromebooks](https://www.chromium.org/chromium-os/chrome-os-systems-supporting-linux)
+    - [ChromeOS Flex Chromebooks](https://support.google.com/chromeosflex/answer/11513094)
 2.  You can install Linux by following [these instructions](https://support.google.com/chromebook/answer/9145439?hl=en)
 
 Once you have successfully met both of these requirements, you should be able to follow along with the Linux instructions throughout the entire curriculum.
 
 </details>
 
+<details markdown="block">
+<summary class="dropDown-header">WSL2</summary>
+
+Using WSL2 is an quick and easy way to get started with using Linux, allowing you to run a Linux distribution from within Windows. WSL2 is available on Windows 10 version 2004 and higher (Build 19041 and higher) and Windows 11.
+
+<div class="lesson-note">
+<h4>WSL2 and Linux instructions</h4>
+Because WSL2 is a full fledged Linux distribution, almost everything that the curriculum teaches about Linux is also applicable to WSL2. In future lessons whenever there are instructions that differ by OS you should follow the Linux instructions, unless the lesson includes WSL2 specific instructions.
+</div>
+
+### Step 1: Installations
+
+#### Step 1.1: WSL2 and Virtual Machine Platform
+
+In order to make WSL2 work we'll first need to ensure both `Windows Subsystem for Linux` and `Virtual Machine Platform` are enabled on your system.
+
+- Search for Control Panel in your applications, and open it
+- Select **Programs**
+- Select **Turn Windows Features on or off** (under the Programs and Features header)
+- Ensure both `Windows Subsystem for Linux` and `Virtual Machine Platform` are toggled on, and if not toggle them on.
+- When prompted, restart your computer
+
+#### Step 1.2: Installing WSL2
+
+- Open PowerShell in administrator mode by searching for it in your applications, right clicking the top option, and then selecting run as administrator. You might get a prompt asking if you want to allow Windows Powershell to make changes to your device: click yes.
+- Enter the following command
+
+~~~powershell
+wsl --install
+~~~
+
+- After a few minutes you'll be prompted to reboot your computer; do so.
+- You should see an open Powershell window, prompting you to enter a username and a password. Your username should be lowercase, but can otherwise be whatever suits you. You'll also need to enter a new password.
+- When entering your password you might notice that you aren't seeing any visual feedback. This is a standard security feature in Linux, and will also happen in all future cases where you need to enter some form of password. Rest assured that you are actually inputting something.
```

Comment:
```suggestion
- When entering your password you might notice that you aren't seeing any visual feedback. This is a standard security feature in Linux, and will also happen in all future cases where you need to enter a password. Just type your password and hit <kbd>Enter</kbd>.
```

nit, but I thought this could be shortened as there aren't many forms of passwords. Do with the last sentence as you please, I thought I'd offer an alternative.